frontend/pages/group/data/foods.vue

```diff
@@ -134,6 +139,11 @@
             :label="$t('data-pages.foods.food-label')"
           >
           </v-autocomplete>
+          <v-checkbox
+            v-model="editTarget.onHand"
+            :label="$t('data-pages.foods.on-hand-checkbox-label')"
+          >
+          </v-checkbox>
         </v-form>
       </v-card-text>
       <template #custom-card-action>
```

Comment:
Can we add a custom display option for the boolean in the `CrudTable` (line 230) similar to how tools does it? It's below this comment, but GH won't let me add a comment there since you didn't modify it.

```html
<template #item.onHand="{ item }">
  <v-icon :color="item.onHand ? 'success' : undefined">
    {{ item.onHand ? $globals.icons.check : $globals.icons.close }}
  </v-icon>
</template>
```

You should be able to insert this right under the label template (line 256)